app/src/main/AndroidManifest.xml

```diff
@@ -40,8 +35,13 @@
             android:name=".ConductorActivity"
             android:configChanges="orientation|keyboardHidden|screenSize"
             android:label="@string/title_activity_conductor"
-            android:theme="@style/FullscreenTheme" />
 
+            android:theme="@style/FullscreenTheme">

```

Comment:
Why did you make the conductor activity the starting activity? It should be the splash.